src/bindgen/library.rs

```diff
@@ -633,8 +634,10 @@ impl Library {
         // Gather only the items that we need for this
         // `extern "c"` interface
         let mut deps = DependencyList::new();
+        let mut oop = MemberFunctions::new();
```

Comment:
Object orientated programming (It is a bad name, should better be something like member_functions)